deps/libsuitesparse.mk

```diff
@@ -3,14 +3,14 @@ include $(SRCDIR)/libsuitesparse.version
 
 ifneq ($(USE_BINARYBUILDER_LIBSUITESPARSE), 1)
 
-LIBSUITESPARSE_PROJECTS := "amd;btf;camd;ccolamd;colamd;cholmod;klu;ldl;umfpack;rbio;spqr"
+LIBSUITESPARSE_PROJECTS := "suitesparse_config;amd;btf;camd;ccolamd;colamd;cholmod;klu;ldl;umfpack;rbio;spqr"
 LIBSUITESPARSE_LIBS := $(addsuffix .*$(SHLIB_EXT)*,suitesparseconfig $(subst ;, ,$(LIBSUITESPARSE_PROJECTS)))
 
 LIBSUITESPARSE_CMAKE_FLAGS := $(CMAKE_COMMON) \
 	  -DCMAKE_BUILD_TYPE=Release \
 	  -DBUILD_STATIC_LIBS=OFF \
 	  -DBUILD_TESTING=OFF \
-	  -DSUITESPARSE_ENABLE_PROJECTS="suitesparse_config;$(LIBSUITESPARSE_PROJECTS)" \
+	  -DSUITESPARSE_ENABLE_PROJECTS=$(LIBSUITESPARSE_PROJECTS) \
```

Comment:
` -DBLAS64_SUFFIX="_64"	  -DSUITESPARSE_USE_64BIT_BLAS=YES` should only be for 64-bit platforms.

Also, we use `libblastrampoline-5` in Yggdrasil for Windows - but I am not sure if that should be done in a source build too.

Reply:
OK, I've add it.